.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature Request 
+about: Add a feature request
+---
+##  Before Logging a feature request
+[ ] Please search this repository's issues to see if it has already been reported. This helps reduce duplicate issues from being created.

```

Comment:
same here:
```
- [ ] I have searched in this repository's issues to see if it has already been reported.
- [ ] This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-edge-ios/security/policy).
```